lib/msal-core/src/telemetry/TelemetryConstants.ts

```diff
@@ -0,0 +1,19 @@
+export const EVENT_NAME_PREFIX: String = "msal.";
+export const EVENT_NAME_KEY: String = "event_name";
+export const START_TIME_KEY: String = "start_time";
+export const ELAPSED_TIME_KEY : String = "elapsed_time";
+
+export const TELEMETRY_BLOB_EVENT_NAMES =  {
+    MsalCorrelationIdConstStrKey: "Microsoft.MSAL.correlation_id",
+    ApiTelemIdConstStrKey: "msal.api_telem_id",
+    ApiIdConstStrKey: "msal.api_id",
+    BrokerAppConstStrKey: "Microsoft_MSAL_broker_app",
```

Comment:
Why do we need Broker APP?

Reply:
we probably do not, once I wire up all of the events, I will be pruning what is unused